contract/src/main/resources/swagger/kafbat-ui-api.yaml

```diff
@@ -2527,7 +2542,7 @@ components:
         partitions:
           type: array
           items:
-            $ref: "#/components/schemas/Partition"
```

Comment:
Could we revert all unnecessary formatting changes? Makes reviewing the PR complicated. Happy to discuss the necessity of formatting changes in a separate PR.

Reply:
I rolled back the formatting changes :)